### PR TITLE
feat(readme): add validations in each PR documentation

### PR DESCRIPTION
**Pull request by cbernat**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,14 +2,34 @@
 
 This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
-## Install
+## Testing in each PR
 
-In the project directory, you can run:
+Each time a PR is made CI is run, to see a full detail check [DockerFile](https://github.com/FromDoppler/doppler-webapp/blob/master/Dockerfile.BUILDS_AND_CDN).
+As a result a temporal version of the code is published into CDN marked with a build number.
+To check the build number in the second check marked in each PR, while hovering in details the version is as marked in the image below.
```
Comment (andresmoschini) [inline README.md:9]: Markdown requires an extra interline for each paragraph.

```suggestion
Each time a PR is made CI is run, to see a full detail check [DockerFile](https://github.com/FromDoppler/doppler-webapp/blob/master/Dockerfile.BUILDS_AND_CDN).

As a result a temporal version of the code is published into CDN marked with a build number.

To check the build number in the second check marked in each PR, while hovering in details the version is as marked in the image below.
```

File: README.md [hunk 2]
```diff
@@ -2,14 +2,34 @@
...
+
+- **Production:** <https://cdn.fromdoppler.com/doppler-webapp/build2962/#/login>
+- **Integration:** <https://cdn.fromdoppler.com/doppler-webapp/int-build2962/#/login>
+- **QA:** <https://cdn.fromdoppler.com/doppler-webapp/qa-build2962/#/login>
+- **Demo\*:** <https://cdn.fromdoppler.com/doppler-webapp/demo-build2962/#/login>
+- **Development\*\*:** <https://cdn.fromdoppler.com/doppler-webapp/demo-build2962/#/login>
+
+*Demo is like a local copy hosted into CDN (it uses doubles and no real data).
+
+**Development is code that points to local Doppler (It is needed to have local Doppler copy running for this to work).
```
Comment (andresmoschini) [inline README.md:23]: What do you think about adding the clarifications within the list

```suggestion
- **Production:** <https://cdn.fromdoppler.com/doppler-webapp/build2962/#/login>

- **Integration:** <https://cdn.fromdoppler.com/doppler-webapp/int-build2962/#/login>

- **QA:** <https://cdn.fromdoppler.com/doppler-webapp/qa-build2962/#/login>

- **Demo:** <https://cdn.fromdoppler.com/doppler-webapp/demo-build2962/#/login>

  "Demo" is like a local copy hosted into CDN (it uses doubles and no real data).

- **Development:** <https://cdn.fromdoppler.com/doppler-webapp/demo-build2962/#/login>

  "Development" is code that points to local Doppler (It is needed to have local Doppler copy running for this to work).
```

**Comment by cbernat** [comment]
@FromDoppler/kreitech FYI

**Pull request merged**